opendebates/tasks.py

```diff
@@ -42,6 +72,24 @@ def update_recent_events():
 
         logger.debug("There are %d entries" % len(entries))
         logger.debug("There are %d votes" % number_of_votes)
+    except:
+        # This task runs too frequently to report a recurring problem every time
+        # it happens.
+        global exception_logged
```

Comment:
I'm not 100% sure how this will work. The first time an exception occurs, it will get logged and the global flag will get set to prevent further logging. When will that flag get unset? Only when we restart celery? Just want to make sure I understand it.